README.md

```diff
@@ -1,2 +1,68 @@
+
 # react-native-shopify
-React Native bridge to Shopify Buy SDK
+
+## Getting started
+
+`$ npm install react-native-shopify --save`
+
+### Including Mobile-BUY-SDK
+
+Include the Shopify Mobile Buy SDK in your project to make it available to the bridge. 
+Follow the instructions on their Github page to get started. For example, 
+the recommended and easiest way for iOS is to install it as a Pod. This project will look 
+for headers in the Pods directory. 
```

Comment:
It works in any of the cases they mentioned in their documentation. The Cocoapods approach is most practical.